### DISCUSSION TITLE
Willkommen!

### DISCUSSION THREAD

**Discussion by tbk303** [bv-hh/bv-hh#6] [Allgemein]
## 👋 Moin!
In diesem Bereich können sich Interessierte rund um BV-HH.de austauschen. Insbesondere:
  * Allgemein diskutieren
  * Fragen stellen
  * Neue Ideen teilen  

Gemeinsam können wir BV-HH.de weiter voranbringen 💪!

**Comment by 2martens** [comment]
Super geile Sache. Habe selber auch schonmal was dazu gemacht (mit Python versteht sich): https://git.2martens.de/2martens/allris-scraper

Nutze das, um mir die Mappen für die Sitzungen herunterzuladen und dann passwortgeschützt abzulegen. Noch schöner wäre es aber natürlich, wenn Allris (was die Software in der nächsten Majorversion offiziell unterstützt) auch OParl 2.0 anbieten und bespielen würde. Siehe [oparl.org](https://oparl.org).

Hatte da auch schon einmal angefangen einen [Java Spring Boot REST-Service](https://git.2martens.de/2martens/oparl-service) zu schreiben, der die Schnittstelle von Oparl serverseitig anbietet und somit als eine Art Adapter zwischen Allris und beliebigen Drittanwendungen stehen könnte. Zum Testen hatte ich mir damals mittels meines Scrapers einen Dump der Vorlagen vom Juni 2020 geholt. Dann ist das aber wieder eingeschlafen.

**Reply by tbk303** [reply]
Ja, auf OParl bin ich auch gespannt, das würde einiges einfacher machen. Ich hab da doch den einen oder anderen miesen Hack drin, um einige Klippen zu umschiffen. Es gibt auch ganz subtile Unterschiede zwischen den Bezirken, die man erst merkt, wenn man wirklich überall einmal Drucksachen gelesen hat.
